core/schemas/__init__.py

```diff
@@ -19,30 +20,76 @@
 )
 
 logger = logging.getLogger(__name__)
+logging.getLogger().setLevel(logging.INFO)
 
 
-def load_entities():
-    logger.info("Registering entities")
-    modules = dict()
-    for entity_file in Path(__file__).parent.glob("entities/**/*.py"):
-        if entity_file.stem == "__init__":
+def register_schemas_types(
+    schema_root_type: str, schema_enum: aenum, type_pattern: re.Pattern
+) -> set:
+    """
+    Register the types of schemas from implementation files
+    :param schema_root_type: The schema root type to work with
+    :param schema_enum: The schema enum to extend
+    :param type_pattern: The pattern to match the types in the schema implementation files
```

Comment:
Please use a more detailed docstring, we also don't use the `:params` notation. What does it return?